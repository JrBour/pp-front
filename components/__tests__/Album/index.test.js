import { shallowMount } from '@vue/test-utils'
import Album from '@/components/Album'


describe('Participant', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallowMount(Album, {
      propsData: {
        id: 1,
        media: 'media.jpg',
        title: 'Media',
        number: 2
      }
    })
  })

  describe('snapshot', () => {
    it('renders properly', () => {
      expect(wrapper.html()).toMatchSnapshot()
    })
  })
})
