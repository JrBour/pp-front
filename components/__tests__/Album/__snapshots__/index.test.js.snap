// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Participant snapshot renders properly 1`] = `
<div class="album">
  <div class="album__cover" style="background-image: url(undefined/media/media.jpg);"></div>
  <p class="album__title">Media</p>
  <p class="album__numberOfPictures">
    2 photos
  </p>
</div>
`;
